pkg/repository/v1/sqlcv1/tasks.sql

```diff
@@ -976,3 +976,101 @@ SELECT
         slot.workflow_run_id
     )
 FROM active_slots slot;
+
+-- name: GetTenantWorkflowStats :many
+WITH queued_tasks AS (
+    SELECT
+        'queued' as status,
+        w.name as workflow_name,
+        unnest(t.concurrency_keys) as concurrency_key,
+        COUNT(*) as count
+    FROM
+        v1_queue_item qi
+    JOIN
+        v1_task t ON qi.task_id = t.id AND qi.task_inserted_at = t.inserted_at
+    JOIN
+        "Workflow" w ON t.workflow_id = w.id
+    WHERE
+        qi.tenant_id = @tenantId::uuid
+        AND t.tenant_id = @tenantId::uuid
```

Comment:
Need to be a bit careful with these if they're not in the pkey - have we checked how this performs on a very large database?